src/organisms/cards/RadioCard/radio_card.module.scss

```diff
@@ -71,6 +73,42 @@
   margin: auto auto ru(-1.5);
 }
 
+// different state based on sections.
+.sectioned {
+  .card {
+    position: relative;
+    padding: ru(2.75) ru(1) ru(1);
+  }
+}
+
+.sections {
+  @include typography-7;
+
+  display: flex;
+  margin: auto ru(-1) ru(-1);
```

Comment:
Do we really need to use negative margins everywhere for this? I think it would be preferable to just remove the padding, instead of canceling it out with margins.

If we have to use negative margins, it would be better to make them use variables that are tied to the padding they are negating so that they will always be in sync.